monkey/infection_monkey/exploit/__init__.py

```diff
@@ -49,6 +59,18 @@ def add_vuln_url(self, url):
     def add_vuln_port(self, port):
         self._exploit_info['vulnerable_ports'].append(port)
 
+    def add_example_cmd(self, cmd):
+        self._exploit_info['executed_cmds']['example'] = cmd
+
+    def add_powershell_cmd(self, cmd):
```

Comment:
remove this function.
You can use only `add_example_cmd` for both.
Check if the command is powershell or not, and save the cmd as a dict containing the actual command, and whether it uses powershell or not.

No matter how we decide to resolve the add_example_cmd thing, this change for powershell will be the same

Reply:
I know this probably doesn't matter here, but keep in mind that solutions like this will negatively affect performance.